README.md

```diff
@@ -1 +1,46 @@
-# Cr-Portfolio_Project
\ No newline at end of file
+# curriculum-cr-training-contact-form
+
+This repo host an initial code for the tasks performed during the Code Reviewers training.
+
+Application is based on this [Figma design](https://www.figma.com/file/t3EJUCAEViw3QasuJLPLVT/Microverse-Student-Potfolio-Templates-Main?node-id=1%3A1471).
+
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+![screenshot](./img/212.png)
+
```

Comment:
[Optional] 
your screenshot does not depict a good design of your webpage, implement the change requested and make another screenshot, it would make you readme file more professional